.gitignore

```diff
@@ -5,3 +5,4 @@ test/
 *.txt
 *.sav
 roms/
+*.xcodeproj
```

Comment:
Shouldn't be an Xcodeproj in there should there?

Reply:
Leftover from old builds I think. We can leave it in the gitignore though.

Reply:
I think I would rather keep the gitignore as small as possible

Reply:
removed